CMakeCommon.cmake

```
@@ -287,6 +286,15 @@ function (GenerateAddOnProject target acVersion devKitDir addOnSourcesFolder add
             BUNDLE TRUE
             LIBRARY_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/\$<CONFIG>"
         )
+        if ((NOT codesignIdentity STREQUAL "") AND (NOT developmentTeamId STREQUAL ""))
+            set_target_properties("${target}" PROPERTIES
+                XCODE_ATTRIBUTE_CODE_SIGN_IDENTITY "${codesignIdentity}"
+                XCODE_ATTRIBUTE_CODE_SIGN_STYLE "Manual"
+                XCODE_ATTRIBUTE_DEVELOPMENT_TEAM "${developmentTeamId}"
+                XCODE_ATTRIBUTE_CODE_SIGN_ENTITLEMENTS "${CMAKE_SOURCE_DIR}/Tools/entitlements.plist"
```

[CMAKE_CURRENT_FUNCTION_LIST_DIR](https://cmake.org/cmake/help/latest/variable/CMAKE_CURRENT_FUNCTION_LIST_DIR.html)

```suggestion
                XCODE_ATTRIBUTE_CODE_SIGN_ENTITLEMENTS "${CMAKE_CURRENT_FUNCTION_LIST_DIR}/entitlements.plist"
```